src/version.cpp

```diff
@@ -37,38 +37,38 @@ const std::string CLIENT_NAME("Satoshi");
 
 // First, include build.h if requested
 #ifdef HAVE_BUILD_INFO
-#    include "build.h"
+#include "build.h"
```

Comment:
I'd prefer if these were indented, pound sign and all (they're enclosed in an if statement, even if it is an ifdef).